bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java

```diff
@@ -591,56 +627,97 @@ private void handleExceptionWhileCreatingUser(Exception ex) {
                 "@text/offline.conf-error-creation-username");
     }
 
+    public boolean registerDiscoveryListener(HueLightDiscoveryService listener) {
```

Comment:
Light and sensor states have to be pushed to the discovery listener. This was done before your changes.

Reply:
Pushing again on registration. Please have a look.